projects/plugins/wpcomsh/package.json

```
@@ -0,0 +1,41 @@
+{
+	"author": "Automattic",
+	"name": "@automattic/jetpack-wpcomsh",
+	"description": "A helper for connecting WordPress.com sites to external host infrastructure.",
+	"homepage": "https://jetpack.com",
+	"version": "3.22.6-alpha",
+	"bugs": {
+		"url": "https://github.com/Automattic/jetpack/labels/[Plugin] Wpcomsh"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/Automattic/jetpack.git",
+		"directory": "projects/plugins/wpcomsh"
+	},
+	"license": "GPL-2.0-or-later",
+	"devDependencies": {
+		"@babel/core": "^7.5.5",
+		"@babel/preset-env": "^7.5.5",
+		"@wordpress/babel-preset-default": "^4.4.0",
+		"babel-jest": "^24.9.0",
+		"dotenv": "^8.1.0",
+		"dotenv-expand": "^5.1.0",
+		"eslint": "^6.2.0",
+		"eslint-config-prettier": "^6.0.0",
+		"eslint-plugin-import": "^2.18.2",
+		"eslint-plugin-jest": "^22.15.1",
+		"jest": "^29.7.0",
+		"lodash": "^4.17.15",
+		"node-fetch": "^2.6.0",
+		"wp-prettier": "^1.1.7"
```

Same as the eslint stuff, wp-prettier is probably not needed here.
```suggestion
		"node-fetch": "^2.6.0"
```